cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -150,7 +162,13 @@ def index_item(item, skip_index=False):
                 skip_child_index = skip_index or \
                     (triggered_at is not None and (triggered_at - item.subtree_edited_on) > reindex_age)
                 for child_item in item.get_children():
-                    index_item(child_item, skip_index=skip_child_index)
+                    item_content_groups = getattr(item, 'content_groups', None)
+                    if item_content_groups:
```

Comment:
Don't really need this conditional here - if we pass through `item_content_groups`, then the check for `if content_groups` will be falsy if `item_content_groups==[]`; so if we feed it on through to the child call, everything should just work :-)